app/assets/javascripts/map.js

```diff
@@ -4,16 +4,26 @@ $(function() {
 
   var createMap = function(latitude, longitude){
     L.mapbox.accessToken = window.MAPBOX_ACCESS_TOKEN;
-    map = L.mapbox.map('map', window.MAPBOX_USER_NAME).
+    map = L.mapbox.map('map', window.MAPBOX_USER_NAME, { zoomControl: false }).
```

Comment:
Didn't catch this earlier, but should be using double quotes here. Also, probably should enable hound for javascript.